Add handling of players who had no matches in a series

drdarina opened the pull request:
Currently the code will break if a series is empty. According to the [paper](http://www.glicko.net/glicko/glicko2.pdf) by Mark Glickmann, in this case only `phi_star` needs to be updated.

Commit:
Add handling of players who had no matches in a series

sublee commented on glicko2.py:
```diff
@@ -136,31 +136,36 @@ def rate(self, rating, series):
+        else:
+            # If the team didn't play in the series, do only Step 6
+            phi_star = math.sqrt(rating.phi ** 2 + rating.sigma ** 2)
+            return self.scale_up(self.create_rating(rating.mu, phi_star, rating.sigma))
```
Would you consider early-return?

```suggestion
        if not series:
            # If the team didn't play in the series, do only Step 6
            phi_star = math.sqrt(rating.phi ** 2 + rating.sigma ** 2)
            return self.scale_up(self.create_rating(rating.mu, phi_star, rating.sigma))

        for actual_score ...
```

drdarina replied:
Accidentally put my reply as commit message, sorry. Absolutely, yes.

sublee replied:
My comment was not for on right the line. I had not to wrap it with `suggest`. Sorry about that. Would you please review the diff again and apply my suggestion correctly?

drdarina committed:
Update glicko2.py

Yes, absolutely. The only reason I didn't go for it was I wanted to be as little invasive as possible.

Co-Authored-By: Heungsub Lee <sub@subl.ee>

Commit:
Implement early return

Commit:
Remove unneccessary else clause

Commit:
Merge branch 'bugfix/fix-zero-div-error-issue-3' of github.com:drdarina/glicko2 into bugfix/fix-zero-div-error-issue-3

Commit:
Clean up merge issues
